questionnaire/routes.test.js

```diff
@@ -266,40 +272,37 @@ describe('POST /questionnaires', () => {
                     data: {
                         type: 'questionnaires',
                         attributes: {
-                            templateName: 'this-does-not-exist',
-                            owner: {
-                                id: 'urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6',
-                                isAuthenticated: true
-                            }
+                            templateName: 'sexual-assault'
                         }
                     }
                 });
             expect(response.body).toHaveProperty('errors');
-            expect(response.body.errors[0].status).toEqual(404);
-            expect(response.body.errors[0].detail).toEqual(
-                'Template "this-does-not-exist" does not exist'
-            );
+            expect(response.body.errors[0].status).toEqual(400);
+            expect(response.body.errors[0].detail).toEqual("should have required property 'owner'");
         });
 
         it('should return status code 201 if owner data is included in the request body', async () => {
-            jest.doMock('./questionnaire-service.js', () =>
-                jest.fn(() => ({
-                    createQuestionnaire: () => {
-                        return {
-                            type: 'questionnaires',
-                            id: '285cb104-0c15-4a9c-9840-cb1007f098fb',
-                            attributes: {
+            jest.isolateModules(() => {
```

Comment:
This isn't necessarily an incorrect usage. It indicates to me that set up and tear down of mocks is not working as expected elsewhere.